src/cronjobs/retrieve-usd-prices.ts

```diff
@@ -95,8 +95,27 @@ async function prepareLegacyTokens(chain: LegacyChain, date: Date, tokensToQuery
       }
     }
 
+    // if ZUSD token price is requested, but subgraph does not return it, add price of DLLR to it.
+    if (
+      tokensToQuery.find((item) => item.address.toLowerCase() === chain.config.zusdToken.toLowerCase()) &&
```

Comment:
you can create a func like we have in dap2 - `areAddressesEqual` and use it there